libs/core/src/components/pds-input/pds-input.tsx

```diff
@@ -256,7 +256,9 @@ export class PdsInput {
         aria-readonly={this.readonly ? 'true' : null}
       >
         <div class="pds-input">
-          <label htmlFor={this.componentId}>{this.label}</label>
+          {this.label &&
+            <label htmlFor={this.componentId}>{this.label}</label>
+          }
```

Comment:
I'm curious if we still want his approach, with how we just changed the checkbox to wrap the input with `label.`. Thoughts?

Reply:
Yes, this approach is still valid. I don't believe I've ever seen a label wrap an input not of type `checkbox`, `radio`. I believe those are the only ones that get that treatment because the text is typically inline with the input on either side. The nesting would cause the need for additional `div`s to have control over layout